composer.json

```diff
@@ -450,9 +454,6 @@
             "drupal/inline_entity_form": {
                 "Add a setting to enable/disable inline editing of existing entities (https://www.drupal.org/project/inline_entity_form/issues/2913571#comment-12811088)": "https://www.drupal.org/files/issues/2018-10-11/ief_edit_existing_2913571_7.patch"
             },
-            "drupal/iframe_resizer": {
-                "Missing license URL (https://www.drupal.org/project/iframe_resizer/issues/3385378)": "https://www.drupal.org/files/issues/2023-09-05/iframe_resizer-missing_license_url-3385378-3.patch"
-            },
```

Comment:
This is a duplicate, we already had that several lines above, so just deleted duplicates here.